### PR TITLE
[Doc] Update *Debezium Connector Guide*

### PR DESCRIPTION
**Pull request by Anonymitaet**

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@tuteng could you please help review? Thank you

File: site2/docs/io-cdc-debezium.md [hunk 1]
```diff
@@ -1,227 +1,281 @@
...
+
+    ## KEY_CONVERTER_CLASS_CONFIG, VALUE_CONVERTER_CLASS_CONFIG
+    key.converter: "org.apache.kafka.connect.json.JsonConverter"
+    value.converter: "org.apache.kafka.connect.json.JsonConverter"
+
+    ## PULSAR_SERVICE_URL_CONFIG
+    pulsar.service.url: "pulsar://127.0.0.1:6650"
+
+    ## OFFSET_STORAGE_TOPIC_CONFIG
+    offset.storage.topic: "offset-topic"
```
Comment (tuteng) [inline site2/docs/io-cdc-debezium.md]: There seems to be a loss of indentation here.

File: site2/docs/io-cdc-debezium.md [hunk 2]
```diff
@@ -1,227 +1,281 @@
...
-  database.server.name: "dbserver1"
-  schema.whitelist: "inventory"
+    ## config for pg, docker image: debezium/example-postgress:0.8
+    database.hostname: "localhost"
+    database.port: "5432"
+    database.user: "postgres"
+    database.password: "postgres"
+    database.dbname: "postgres"
+    database.server.name: "dbserver1"
+    schema.whitelist: "inventory"
```
Comment (tuteng) [inline site2/docs/io-cdc-debezium.md]: There seems to be a loss of indentation here.

**Comment by Anonymitaet** [comment]
@tuteng I've incorporated all your comments, could you PTAL?

**Comment by tuteng** [comment]
LGTM

**Comment by tuteng** [comment]
run integration tests

**Pull request merged**